CHANGELOG.md

```diff
@@ -118,7 +120,8 @@ Backport <https://github.com/itchysats/itchysats/pull/924> in an attempt to fix
 
 Initial release for mainnet.
 
-[Unreleased]: https://github.com/itchysats/itchysats/compare/0.4.12...HEAD
```

Comment:
❓ Why not `0.4.13`?

Reply:
We decided for the patch release to go for 0.4.12.1 
Why is this file changed in the first place? 🧐

Reply:
> I don't disapprove going into the direction of letting the maker come up with the fee for now, but I think this solution will cause trouble. Please have a look at the comments.

What trouble do you see? Can you elaborate please.

Reply:
This commit is not part of this PR. 
It shows up because I branched away from the tag `0.4.12.1` and not the branch `0.4.12.1`. 
I think this commit should belong into the branch `0.4.12.1`.

cc @klochowicz